library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -359,6 +359,7 @@
             DownloadContract.Downloads.COLUMN_CURRENT_BYTES + " AS " + COLUMN_BYTES_DOWNLOADED_SO_FAR,
             DownloadContract.Downloads.COLUMN_BATCH_ID,
             DownloadContract.Downloads.COLUMN_EXTRA_DATA,
+            DownloadContract.Downloads.COLUMN_NOTIFICATION_EXTRAS,
```

Comment:
This wasn't part of the _translated_ cursor, so the cursor we've been given back when querying did not contains this column.